azure-quantum/azure/quantum/aio/target/quantinuum.py

```diff
@@ -0,0 +1,42 @@
+##
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+##
+from typing import Any, Dict
+
+from azure.quantum.aio.target.target import Target
+from azure.quantum.aio.job.job import Job
+from azure.quantum.aio.target import Honeywell
+
+
+class Quantinuum(Honeywell):
+    """Quantinuum target."""
+
+    async def submit(
+        self,
+        circuit: str,
+        name: str = "quantinuum-job",
+        num_shots: int = None,
+        input_params: Dict[str, Any] = None,
+        **kwargs
+    ) -> Job:
+        """Submit a Quantinuum program (QASM format)
+
+        :param circuit: Quantum circuit in Quantinuum (formerly Honeywell) QASM format
```

Comment:
Similarly here.

Reply:
I replaced the documentation in other places too.